connector/google/google.go

```diff
@@ -23,7 +24,7 @@ type Config struct {
 	// Optional list of whitelisted domains when using Google
 	// If this field is nonempty, only users from a listed domain will be allowed to log in
 	HostedDomains []string `json:"hostedDomains"`
-	ServiceAccountCred  string `json:"serviceAccountCred"`
+	filePath  string `json:"serviceAccountCred"`
```

Comment:
Should be capitalized ie, `FilePath`

Reply:
Would make more since also to name this something like `ServiceAccountCredentialPath`